Sources/UIFont.swift

```diff
@@ -33,6 +33,10 @@ open class UIFont {
     }
 
     public init?(name: String, size: CGFloat) {
+        guard UIScreen.main != nil else {
+            return nil
+        }
```

Comment:
nit: we could also do `guard let mainScreen = UIScreen.main ...` and use it below.